build.sbt

```diff
@@ -0,0 +1,56 @@
+inThisBuild(Seq(
+  organization := "org.portable-scala",
+  version := "1.0.0-SNAPSHOT",
+
+  scalaVersion := "2.12.3",
+  scalacOptions ++= Seq("-deprecation", "-feature", "-encoding", "UTF-8"),
+
+  crossSbtVersions := Seq("1.0.2", "0.13.16"),
+
+  homepage := Some(url("https://github.com/portable-scala/sbt-platform-deps")),
+  licenses += ("BSD 3-Clause",
+      url("https://github.com/portable-scala/sbt-platform-deps/blob/master/LICENSE")),
+  scmInfo := Some(ScmInfo(
+      url("https://github.com/portable-scala/sbt-platform-deps"),
+      "scm:git:git@github.com:portable-scala/sbt-platform-deps.git",
+      Some("scm:git:git@github.com:portable-scala/sbt-platform-deps.git"))),
+))
+
+lazy val `sbt-platform-deps` = project.in(file(".")).
+  settings(
+    sbtPlugin := true,
+
+    scriptedLaunchOpts ++=
+      Seq("-Xmx1024M", "-Dplugin.version=" + version.value),
```

Comment:
scripted reports

    [error] Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m

Is this expected? (seems to work fine, just a bit strange its an "error").

Reply:
I will look into it. Maybe we don't need that specific option anyway.